components/automate-ui/src/app/ui.component.ts

```diff
@@ -44,20 +44,25 @@ export class UIComponent implements OnInit {
 
   constructor(
     private store: Store<NgrxStateAtom>,
-    private router: Router
+    private router: Router,
+    private route: ActivatedRoute
   ) {
     this.notifications$ = store.select(notificationState);
   }
 
   ngOnInit(): void {
     this.router.events.pipe(
-        filter(event => event instanceof ActivationStart)
+        filter(event => {
+          return event instanceof ActivationStart;
+        })
     ).subscribe((event: any) => {
       this.renderNavbar = event.snapshot.data.hideNavBar
         ? false
         : true;
     });
 
+    this.renderNavbar = this.route.snapshot.data.hideNavBar;
```

Comment:
Two issues. First is polarity:
```suggestion
    this.renderNavbar = !this.route.snapshot.data.hideNavBar;
```
Second, I don't have a solution for as yet... but the `data` is empty; it is not returning the static route data that it seems like it should be. That is why no pages are getting the top nav bar, as @susanev observed.

Reply:
@msorens Tested more and value is available in constructor with first ActivationEnd, updated.